css/styles.css

```
@@ -1,9 +1,331 @@
+@font-face {
+    font-family: "Inter";
+    font-style: normal;
+    font-weight: 400;
+    src: url(../fonts/Inter-400.woff2);
```

```suggestion
    src: url("../fonts/Inter-400.woff2");
```
Добавь кавычки для всех `url`